project_report/__init__.py

```diff
@@ -177,7 +183,27 @@ def enquiry_number(self):
 
     @property
     def number_quoted_samples(self):
-        return self.project.udf.get('Number of Quoted Samples', '')
+        return self.project.udf.get('Number of Quoted Samples', '')\
+
+    @property
+    def customer_name(self):
+        name = ''
+        if self.project.researcher:
+            if self.project.researcher.lab:
+                name = self.project.researcher.lab.name
```

Comment:
Is there a reason this is this a nested 'if' instead of just 'if self.project.researcher.lab.name'?

Reply:
I guess this is in case `researcher.lab` is empty, where it would try to call `.name` on a NoneType field. This shouldn't happen though, because a Lab is supposed to be a list of researchers, so every researcher should have a lab.

Reply:
I've removed the extra test